lua/starfall/libs_cl/render.lua

```diff
@@ -933,6 +927,8 @@ function render_library.drawBlurEffect(blurx, blury, passes)
 	passes = math.Clamp(blurx, 0, 100)
 
 	local rt = render.GetRenderTarget()
+	local w, h = render_library.getGameResolution()
+	local aspectRatio = ScrW() / ScrH()
```

Comment:
Should be w / h right? Also I would avoid using render_library function since users will have control over that.

Reply:
Got it. Should be fixed now